doc/changelog.rst

```diff
@@ -1,6 +1,35 @@
 Changelog
 =========
 
+0.5.0 (unreleased)
+--------------------
+
+This HCIPy release provides further speed improvements for Fourier transforms, among some other minor improvements and bugfixes. See the list of all changes below for a summary. This release can now also be installed with Conda. We dropped support for Python 3.6. This version supports Python 3.7+.
+
+List of major changes
+~~~~~~~~~~~~~~~~~~~~~
+
+* Further speed improvements using NumExpr and in-place operations (`@ehpor <https://github.com/ehpor>`__).
+* Autodetect FFTs when doing general Fourier transforms (`@ehpor <https://github.com/ehpor>`__)
+* Added file IO for Field, Grid and ModeBasis (`@ehpor <https://github.com/ehpor>`__)
+* Allow the use of 32bit floating point Wavefronts (`@ehpor <https://github.com/ehpor>`__)
+* Added the LUVOIR-B, GMT, TMT, Hale, HabEx, Hubble, JWST and VLTI telescope apertures (`@ivalaginja <https://github.com/ivalaginja>`__, `@syhaffert <https://github.com/syhaffert>`__ & `@ehpor <https://github.com/ehpor>`__)
```

Comment:
let's leave the VLTI for the next release. I think the ELT was also added in this release.

Reply:
Agreed about VLTI. I might have missed ELT. Let me check.

Reply:
Yep, I missed the ELT. I don't know how, since it's extremely large. Fixed.

Reply:
Yeah it was part of my major PR that included multiple things.

Reply:
Makes sense. I mostly looked at the names of merged PRs.

Reply:
One more reason to automate the writing of release notes, tbh.